src/index.html

```diff
@@ -10,8 +10,175 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="info-form"
+    >
+      <fieldset class="field-block">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="lname"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="fname"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number" name="number"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="bday"
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="checkbox"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="field-block">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="field-block">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <label>
+           Do you love cats?
+            <input
+              type="radio"
+              name="cats"
+              value="Yes"
+            >Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="No"
+            >No
```

Comment:
I guess you missed fixing this one 👀 
```suggestion
            >
              No
```